go.mod

```diff
@@ -1,13 +1,186 @@
 module github.com/pulumi/kube2pulumi
 
-go 1.16
+go 1.19
```

Comment:
Why this change -- are there Go 1.19 language features in use?

Reply:
All of our centrally-managed GHA workflows are using 1.19 now. We have to be more careful for sdk modules, but normally track pretty close to latest for binary deps.

Reply:
What do you mean by "using" here? The declaration in go.mod gives the version of the language required by the module. There's generally no need for it to be consistent, or the latest, because newer releases support older versions of the language.

Reply:
I mean that the workflows use Go 1.19, so that's what we're building binaries and testing with. It also affects the `go.mod` dependencies, since the import handling has improved quite a bit since modules were introduced.

Reply:
> It also affects the go.mod dependencies, since the import handling has improved quite a bit since modules were introduced.

OK, I see, using `go 1.19` changes what it puts in go.mod. (The pinning would have worked with go 1.16 too! But I shall get out the way of progress :shrug:)